dpgen2/exploration/render/traj_render_lammps.py

```diff
@@ -74,6 +79,11 @@ def _load_one_model_devi(self, fname, model_devi):
         model_devi.add(DeviManager.MAX_DEVI_F, dd[:, 4])  # type: ignore
         model_devi.add(DeviManager.MIN_DEVI_F, dd[:, 5])  # type: ignore
         model_devi.add(DeviManager.AVG_DEVI_F, dd[:, 6])  # type: ignore
+        # assume the 7-9 columns are for MF
+        if dd.shape[1] >= 10:  # type: ignore
```

Comment:
It seems to be better to judge from the lammps input if the spin model deviation is expected to be output or not. 
you may implement a method like `self.has_spin(self.lammps_input)`. 
If the input implies spin model deviation output, then the `shape[1]` of `dd` must be 9 otherwise must be 6.